rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,71 @@
+
+**@fluentui/react-cards** is a weird case since there's an old version (built on @uifabric/foundation createComponent patterns) but it never officially graduated from experimental status and isn't exported from the suite. It also has a "next" folder where the converged version is currently being worked on.
+
+**@fluentui/react-date-time** is what used to be the @uifabric/date-time package. It's a rewrite (mostly by OWA folks, started awhile ago) of the original OUFR Calendar and DatePicker, but still using older patterns. New for v8, the Calendar and DatePicker from this package are exported from @fluentui/react by default.
+
+**@fluentui/react-focus**
+Current version of FocusZone with most of the work needed to converge with v0's FocusZone. The things we're still differing in are some default values and things that are pulled from the theme.
+
+**@fluentui/react-tabs** (Pivot)
+These default-export an old version of the component, which is re-exported by `@fluentui/react`. Work on a new/converged version is happening under `src/next`.
+
+**@fluentui/react-checkbox**
+**@fluentui/react-link**
+**@fluentui/react-slider**
+**@fluentui/react-toggle**
+These default-export a non-converged version of the component, which is re-exported by `@fluentui/react`. Work on a new/converged version is happening under `src/next`. The difference between these and the previous group is that the "current" version has been re-written in a pattern more similar to the new pattern for converged components.
+
+**@fluentui/react-avatar**
+**@fluentui/react-flex**
+**@fluentui/react-image**
+**@fluentui/react-text**
+These packages only contain work on the converged component versions (in varying states of active work and completeness).
+
+**@fluentui/react-charting** has active work ongoing but is owned by a different team and doesn't really follow any of the newer patterns we're working on. (Once we establish a pattern for "high-value component" repos, it would be worth considering whether to split this out.)
+
+**@fluentui/react-experiments** is a weird legacy package which we don't have a good story for moving forward, and probably don't recommend (certainly don't promote) for use except by authoring teams of the individual components. It's a mix of components in active development or heavy use by partners, old trials of previous patterns we should definitely delete, and things of unknown status or abandoned. [More discussion here.](/xLNsv7leTLu9rANVsWf1xQ)
+
+## Problem statement
+
+<!--
+Why are we making this change? What problem are we solving? What do we expect to gain from this?
+
+This section is important as the motivation or problem statement is indepenent from the proposed change. Even if this RFC is not accepted this Motivation can be used for alternative solutions.
+
+In the end, please make sure to present a neutral Problem statement, rather than one that motivates a particular solution
+-->
+
+## Detailed Design or Proposal
```

Comment:
Will work on filling this in later today based on https://hackmd.io/384p4fw4RmeHWPUfJ34xFQ